module/VuFindTheme/src/VuFindTheme/View/Helper/ConcatTrait.php

```diff
@@ -291,6 +341,21 @@ protected function outputInOrder($indent)
     }
 
     /**
+     * Check if a file is minifiable i.e. does not have a pattern that denotes it's
+     * already minified
+     *
+     * @param string $filename File name
+     *
+     * @return bool
+     */
+    protected function isMinifiable($filename)
+    {
+        $basename = basename($filename);
+        return strpos($basename, '.min.js') === false
```

Comment:
Would it be simpler to simply check for ".min." anywhere in the filename, or are you worried that is too generic? Another option might be to do a regular expression match on `\.min\.(js|css)`. Using a regular expression is slightly more concise and might also open the door to making the pattern configurable through an .ini setting in the future if there is a need to do so.